bool is_person
bool is_known_person
int64 id
float32 confidence 
